setup.py

```diff
@@ -8,12 +8,6 @@
     'pytest-xdist==2.2.1'
```

Comment:
This file should be reverted

Reply:
Oh, the intention was to modify line 44 only. It seems required to run it under Python 3.9.4, so I left it for now... let me know.